packages/backend/src/util/request.test-old.ts

```diff
@@ -0,0 +1,86 @@
+// import { checkCrossOrigin } from './request';
```

Comment:
Safe to delete?

Reply:
These are unit tests that we will need to rewrite.